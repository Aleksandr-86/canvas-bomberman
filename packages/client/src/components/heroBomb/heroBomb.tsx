import heroBomb from '../../assets/images/heroBomb.png'

export const HeroBomb = () => {
  return (
    <img
      src={heroBomb}
      width={426}
      height={426}
      alt="Герой символизирующий ошибку"
      draggable={false}
    />
  )
}
