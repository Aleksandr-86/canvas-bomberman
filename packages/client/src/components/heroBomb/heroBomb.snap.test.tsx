import * as renderer from 'react-test-renderer'
import { HeroBomb } from './heroBomb'

describe(`Snapshot test component HeroBomb`, () => {
  test(`Should correct render (without props)`, () => {
    const tree = renderer.create(<HeroBomb />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
