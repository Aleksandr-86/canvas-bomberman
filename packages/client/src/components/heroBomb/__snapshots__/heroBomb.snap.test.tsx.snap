// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test component HeroBomb Should correct render (without props) 1`] = `
<img
  alt="Герой символизирующий ошибку"
  draggable={false}
  height={426}
  width={426}
/>
`;
